### PR TITLE
Contenu - Mise à jour des décisions de conception

### PR DESCRIPTION
**Pull request by duboisp**

### CODE REVIEW

File: site/pages/index-fr.hbs [hunk 1]
```diff
@@ -48,19 +48,18 @@
...
 			<ul>
-				<li><a hreflang="en" lang="en" href="https://wet-boew.github.io/wet-boew-documentation/decision/1.html">DD-1: Use SVG with img element instead of object</a></li>
-				<li><a hreflang="en" lang="en" href="https://wet-boew.github.io/wet-boew-documentation/decision/2.html">DD-2: Browser supported</a></li>
-				<li><a hreflang="en" lang="en" href="https://wet-boew.github.io/wet-boew-documentation/decision/3.html">DD-3: WET API/Blueprint</a></li>
-				<li><a hreflang="en" lang="en" href="https://wet-boew.github.io/wet-boew-documentation/decision/4.html">DD-4: Basic HTML mode and progressive enhancement</a></li>
-				<li><a hreflang="en" lang="en" href="https://wet-boew.github.io/wet-boew-documentation/decision/5.html">DD-5: Repository migration</a></li>
-				<li><a hreflang="en" lang="en" href="https://wet-boew.github.io/wet-boew-documentation/decision/6.html">DD-6: Versioning</a></li>
-				<li><a hreflang="en" lang="en" href="https://wet-boew.github.io/wet-boew-documentation/decision/7.html">DD-7: Experimental feature</a></li>
+				<li><a href="https://wet-boew.github.io/wet-boew-documentation/decision/1-fr.html">DD-1: Use SVG with img element instead of object</a></li>
```
Comment (GormFrank) [inline site/pages/index-fr.hbs]: Put the titles in French please Pierre.

**Review by GormFrank — CHANGES_REQUESTED** [review]

**Comment by GormFrank** [comment]
Pre-approved during technical review as long as latest changes are made.

File: site/pages/index-en.hbs [hunk 2]
```diff
@@ -48,18 +48,17 @@
 	<div class="row">
 		<div class="col-md-6">
 			<h3>Design decision</h3>
-			<p><em>(last updated: <time datetime="2019-12-23">December 2019</time>)</em></p>
+			<p><em>(last updated: <time datetime="2020-01-22">January 2020</time>)</em></p>
```
Comment (GormFrank) [inline site/pages/index-en.hbs]: 22 in content, not just in attribute

**Review by GormFrank — CHANGES_REQUESTED** [review]

**Comment by duboisp** [comment]
Todo: include the propose DD-11

**Review by GormFrank — APPROVED** [review]

**Pull request merged**